src/GraphQL.Conventions/GraphQL.Conventions.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>GraphQL Conventions for .NET</Description>
-    <VersionPrefix>7.2.1-preview</VersionPrefix>
+    <VersionPrefix>7.4.0</VersionPrefix>
```

```suggestion
    <VersionPrefix>7.4.0-preview</VersionPrefix>
```
The proper version number is set during release.  This really doesn't matter unless you're using a preview build.

---

updated